generators/app/index.js

```diff
@@ -248,7 +255,7 @@ module.exports = class extends Generator {
         }
 
         if (oConfig.codeassist) {
```

Comment:
Does the code assist make sense when TypeScript is enabled? Or is there and ExOr somewhere?

Reply:
BTW: variable has a typo `packge` or by intention?

Reply:
Good point, I am not asking the code assist question anymore if typescript is enabled.
https://github.com/ui5-community/generator-ui5-project/pull/58/commits/696df39fb35c8b3fb42aa9e7f0dda67d5f73a58e

Reply:
`packge` was there before, my assumption is that someone did it intentionally so that eslint doesn't complain about using keywords as variable names.